_posts/2021-12-13-why-we-should-rewrite-enzymejs-to-rtl-and-how-to-do-that.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: post
+title: "Why should we rewrite enzyme.js to react-testing-library and how to do that?"
+author: [magdalena.mazur]
+tags: [tech, frontend, javascript, jest, enzyme, react-testing-library, rtl, react]
+---
+## Introduction
+Everyone repeats like a mantra that tests are an indispensable element of development work. Is there something about it? Well, I need to admit that as a developer, I rather often want to skip the test writing stage. I assume I'm not the only one. I am aware that it's a mistake, even with testers on board. Effective and efficient testing of your own code leads to catching bugs at the moment of creating a functionality, as well as when changing something already existing. It cannot be questioned. Sometimes tests also help to understand how some long-unused functionality or component works. And that's a small bonus too.
```

Comment:
I would stick to one version: I am or I'm

Reply:
I changed everywhere to I'm